### PR TITLE
story/VOGRE-12

### PR DESCRIPTION
**Pull request by Girik1105**
# Guidelines for Pull Requests

If you haven't yet read our code review guidelines, please do so, You can find them [here](https://diging.atlassian.net/wiki/spaces/DIGING/pages/2256076801/Code+Review+Guidelines).

Please confirm the following by adding an x for each item (turn `[ ]` into `[x]`).

- [x] I have removed all code style changes that are not necessary (e.g. changing blanks across the whole file that don’t need to be changed, adding empty lines in parts other than your own code)
- [x] I am not making any changes to files that don’t have any effect (e.g. imports added that don’t need to be added)
- [x] I do not have any sysout statements in my code or commented out code that isn’t needed anymore
- [x] I am not reformatting any files in the wrong format or without cause. 
- [x] I am not changing file encoding or line endings to something else than UTF-8, LF
- [x] My pull request does not show an insane amount of files being changed although my ticket only requires a few files being changed
- [ ] I have added Javadoc/documentation where appropriate
- [x] I have added test cases where appropriate
- [x] I have explained any part of my code/implementation decisions that is not be self-explanatory

## Please provide a brief description of your ticket 
This ticket requires to show the user all files associated with a specific text in citesphere and allow the user to import any inidivudal file the user would like.

# Multiple texts for a Citesphere entry

Description:
Citesphere entries can have more than one text attached to them (right now I think the first one will be used?).
The text table should list all the file names that are attached to a record, and then instead of an “Import”  button, there should be an import icon next to each file.

https://diging.atlassian.net/browse/VOGRE-12?atlOrigin=eyJpIjoiNWJjMWNmMGJiYWYyNDdmMzkyYmIyZWMyMGZiMTA0MTEiLCJwIjoiaiJ9

## Are there any other pull requests that this one depends on?
No
  
## Anything else the reviewer needs to know?

I Updated the `repository_text_import` view to support file-specific imports by using `file_id` as part of the unique identifier (`uri`). This change ensures that each file import, even for the same `text_key`, creates or updates a distinct `Text` object. The view now returns `JsonResponse`, providing error messages and a `redirect_url` for successful imports, since now all files are being rendered via AJAX.

### CODE REVIEW

File: repository/managers.py [hunk 1]
```diff
@@ -69,6 +108,7 @@ def item(self, groupId, itemId):
 
         if response.status_code == 200:
             item_data = response.json()
+            print(item_data)
```
Comment (jdamerow) [inline repository/managers.py:111]: remove

**Review by jdamerow — CHANGES_REQUESTED** [review]
Not sure, where this is coming from but I get this error:
`Failed to retrieve Giles document details: Invalid URL 'Noneapi/v2/resources/files/FILEnuoF0dESGftl/content/': No schema supplied. Perhaps you meant http://Noneapi/v2/resources/files/FILEnuoF0dESGftl/content/?`
(when I try to load a text)

**Comment by Girik1105** [comment]
> Not sure, where this is coming from but I get this error: `Failed to retrieve Giles document details: Invalid URL 'Noneapi/v2/resources/files/FILEnuoF0dESGftl/content/': No schema supplied. Perhaps you meant http://Noneapi/v2/resources/files/FILEnuoF0dESGftl/content/?` (when I try to load a text)

I missed adding the `GILES_ENDPOINT` in the env-example. I just updated the example file. Once you add the giles geco staging endpoint in env it should resolve the issue!

**Comment by jdamerow** [comment]
There don't seem to be any commit addressing the PR comments.

**Comment by Girik1105** [comment]
My latest commits are not showing up on this PR, I'll just create a new PR.

**Pull request closed without merging**